test/e2e/framework/util.go

```diff
@@ -3920,6 +3920,29 @@ func WaitForControllerManagerUp() error {
 	return fmt.Errorf("waiting for controller-manager timed out")
 }
 
+func CheckForControllerManagerHealthy() error {
+	var ps string
```

Comment:
Can you name this variable to what information you're getting out of 'docker ps'? Is it the PID?